src/Dlp/V2beta1/Gapic/DlpServiceGapicClient.php

```diff
@@ -183,13 +166,54 @@ private static function getLongRunningDescriptors()
 
     private static function getGapicVersion()
     {
-        if (file_exists(__DIR__.'/../VERSION')) {
-            return trim(file_get_contents(__DIR__.'/../VERSION'));
-        } elseif (class_exists('\Google\Cloud\ServiceBuilder')) {
-            return \Google\Cloud\ServiceBuilder::VERSION;
-        } else {
-            return;
+        if (!self::$gapicVersionLoaded) {
+            if (file_exists(__DIR__.'/../VERSION')) {
+                self::$gapicVersion = trim(file_get_contents(__DIR__.'/../VERSION'));
+            } elseif (class_exists('\Google\Cloud\Version')) {
```

Comment:
I don't think that's possible, because if the class *doesn't* exist, we can't use `Version::class`

Reply:
I'm pretty sure it will work. Test the following:

```php
var_dump(class_exists(IJustMadeThisUp::class));
```

ClassName::class just evaluates to a string.

Reply:
You are totally correct, I will update